terraform/templates/terraform.tfplan.gotmpl

```diff
@@ -5,7 +5,7 @@
 <details><summary>Expand to view details</summary><p>
 
 ```
-{{file.Read "/temp/tfplan.txt"}}
+{{file.Read "/temp/.terraform/tfplan.txt"}}
```

Comment:
@kobrikx why do we need to change there?

Reply:
Otherwise it can't find the file.